.github/workflows/publish.yml

```diff
@@ -94,3 +94,6 @@ jobs:
         env:
           GHR_PATH: artifacts/
           GITHUB_TOKEN: ${{ secrets.CREATE_PR_TOKEN }}
+
+      - name: Monitor Release
+        run: node .github/workflows/scripts/monitor.js
```

Comment:
So we don't use Datadog anymore and fail this action if the monitor fails, right?

Reply:
Correct. The idea of uploading to datadog before was to use this as sort of a makeshift alert for signalling but it didn't prove to be super useful afaik.

Reply:
ahh, I think I was misled by the Copilot-authored PR message.